web/README.md

```diff
@@ -11,11 +11,12 @@ Prerequisites:
 - Ensure you have `NodeJS` and `pnpm` installed locally (`brew install pnpm`)
 - Run `pnpm install` in both the web and mockserver directories
 
-1. Start the mockserver: `yarn run mockserver`
-2. Run app in another tab: `yarn develop`
+1. Start the mockserver: `pnpm run mockserver`
+2. Run app in another tab: `pnpm dev`
```

Comment:
We don't have the command for starting mockerserver from web folder anymore, so let's update this: 
```suggestion
1. Start the mockserver (inside `mockserver` folder): `pnpm run start`
2. Run app in another terminal (inside `web` folder): `pnpm dev`
```

Reply:
I updated the scripts so it works mockserver also works from web (like before).

Reply:
Oh, didn't see you added it - nevermind then 😄